src/api/server.tsx

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+
+const TODO_LIST_URL = "http://127.0.0.1:8000/todo-list/";
+const ADD_TODO_URL = "http://127.0.0.1:8000/todo";
```

Comment:
[Enums](https://www.typescriptlang.org/docs/handbook/enums.html) are pretty cool. You can group together related constants.